server/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+services:
+  nolongerevil:
+    container_name: nolongerevil
+    image: nolongerevil
+    build: .
+    restart: unless-stopped
+    ports:
```

Comment:
Same comment as above about ports from .env or ports from install.sh 

Idea: install.sh generates a .env file, docker-compose or Dockerfile read from .env

Reply:
I submitted https://github.com/bengalaviz/NoLongerEvil-Thermostat/pull/1 to address this